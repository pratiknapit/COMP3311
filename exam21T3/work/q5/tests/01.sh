python3 ./q5 Apartment 1000000 2 1 0
