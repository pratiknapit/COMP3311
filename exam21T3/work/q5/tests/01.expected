#46423: 18/9 Smith Street, Kingsford 2032, 2br, 1ba, 1car, $786000
#45859: 20/75 North Benvenue Street, Kingsford 2032, 2br, 1ba, 0car, $895000
#46570: 17/19 Woodley Street, Alexandria 2015, 2br, 1ba, 0car, $957000
#46829: 12/51 McCauley Street, Alexandria 2015, 2br, 1ba, 0car, $968000
#46639: 14/34 Meeks Street, Kingsford 2032, 2br, 1ba, 0car, $1000000
